tests/unit/amo/components/TestCollectionManager.js

```diff
@@ -349,6 +363,51 @@ describe(__filename, () => {
     expect(state.description).toEqual(secondCollection.description);
   });
 
+  it('renders editable collection add-ons', () => {
+    const extAddon1 = { ...fakeAddon, name: 'uBlock' };
```

Comment:
I'm guessing `ext` stands for `extension`–I think we can just write `extension1` and `extension2`. That's clearer and isn't confusing as to whether this is an external add-on (type) or an extension add-on 😄

Reply:
(Or if it should be `externalAddon` then I think it should be fully labelled as such; the short variable names are usually for line-length but I think they make things less obvious/readable/etc.